python/mxnet/ndarray/contrib.py

```diff
@@ -18,9 +18,76 @@
 # coding: utf-8
 # pylint: disable=wildcard-import, unused-wildcard-import
 """Contrib NDArray API of MXNet."""
+import math
+from ..context import current_context
+from ..random import uniform
 try:
     from .gen_contrib import *
 except ImportError:
     pass
 
-__all__ = []
+__all__ = ["rand_log_uniform"]
+
+def rand_log_uniform(true_classes, num_sampled, range_max, ctx=None):
```

Comment:
I think it should not be called as `rand_log_uniform` because `LogUniform` has a specific meaning. Should be called something like rand_zipfian, or log_uniform_candidate_sampler like in TF.